### DISCUSSION TITLE
Interactive graph of interconnected concepts, ideologies, contextual resources on /guide page

### DISCUSSION THREAD

**Discussion by olarubaj** [rufuspollock/awesome-crypto-critique#146] [Ideas]
> A raw list of concepts/definitions in my opinion doesn't really encourage exploration of the topic. I'm also not sure if there is any particular order in which they occur within each group? Even though the concepts are interlinked there is no way to see how byt looking at the list = there is no "why" behind e.g. "money laundering" concept -> why is it brought up when talking about the web3? There is no context to the concepts.

## Possible solution

- a graph with clickable nodes -> more important ones (i.e. nodes with more links) would be bigger or have different colors or sth.
- this way users would naturally start exploring web3 topics by:
1) starting from the most significant concepts
2) or starting from the concepts of their interest
3) without going into much detail they could see what other concepts are connected to a given concept or in what sources (i.e. podcast episodes) a given concept was mentioned
4) nodes with Life Itself articles/podcasts (contextual concepts) could be marked differently

- if not a graph then maybe at least for now concepts could be represented in a grid of tiles rather than long scrollable list of words

**Comment by rufuspollock** [comment]
@olarubaj really like this idea and something we can look into in the future!

**Reply by theo-cox** [reply]
+1 to Rufus. Great idea, but will be resource intensive so let's get the lower hanging fruit in the other issues rolled out first
